### PR TITLE
Video - adding theme doc

### PR DESCRIPTION
**Pull request by ShimiSun**
#### Where should the reviewer start?
adding theme docs for the video component

#### Any background context you want to provide?
@MikeWalrath let's tag this one on the issue you are filing
#### Do the grommet docs need to be updated?
done + auto
#### Should this PR be mentioned in the release notes?
nop
#### Is this change backwards compatible or is it a breaking change?
 backwards compatible

### CODE REVIEW

File: src/js/components/Video/doc.js [hunk 1]
```diff
@@ -31,3 +31,36 @@ export const doc = Video => {
 
   return DocumentedVideo;
 };
+
+export const themeDoc = {
+  'dark': {
+    description: 'dark color defenition to consider when the Video renders the background color of the scrubber',
```
Comment (ericsoderberghp) [inline src/js/components/Video/doc.js]: "definition"

Comment (ShimiSun) [inline src/js/components/Video/doc.js]: not sure about this one (desc) @ericsoderberghp , let me know what you think.

Comment (ericsoderberghp) [reply src/js/components/Video/doc.js]: What exactly is this? I didn't see a reference to 'dark' from a theme in any of the video files. Also, where in the theme is this? It doesn't seem like it is in `global` or `video`.

Comment (ShimiSun) [reply src/js/components/Video/doc.js]: @ericsoderberghp it's in normalizeColor() https://github.com/grommet/grommet/blob/ec03444fcd59101b8bb804d5119e29a38fd099ce/src/js/utils/colors.js#L6, I wasn't sure about it either. Its used for the video's background color

File: src/js/components/Video/doc.js [hunk 2]
```diff
@@ -31,3 +31,31 @@ export const doc = Video => {
 
   return DocumentedVideo;
 };
+
+export const themeDoc = {
+  'global.colors': {
+    description: 'The background color of the video scrubber.',
+    type: 'string',
+    defaultValue: 'light-4',
```
Comment (ericsoderberghp) [inline src/js/components/Video/doc.js]: I don't think it's quite accurate to say that the default value for `global.colors` is "light-4" or a type string. I think you are trying to say that the color of the Video controls is 'light-4'. Maybe we should add this to the theme as `video.controls.color`?

Comment (ShimiSun) [reply src/js/components/Video/doc.js]: Yes, I agree, I'll refactor the reference of the hardcoded `light-4` to be taken from the theme.

**Comment by ShimiSun** [comment]
@ericsoderberghp not sure what's up with Travis

File: src/js/components/Video/StyledVideo.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ const headStyle = css`
     content: '';
     height: 100%;
     width: ${props => props.theme.global.edgeSize.xsmall};
-    background: ${props => normalizeColor('light-4', props.theme)};
+    background: ${props => normalizeColor(props.theme.video.scrubber.background, props.theme)};
```
Comment (ericsoderberghp) [inline src/js/components/Video/StyledVideo.js]: I think this property should be called `video.scrubber.color`. While we are using the `background` CSS property to show it, this is on the "head" of the scrubber, which should be the "color" of the scrubber.

**Pull request merged**